spa/src/pages/ConfigSteps/OrgsContainer/index.tsx

```diff
@@ -103,66 +81,68 @@ const OrganizationsList = ({
 		}
 	};
 	return (
-		<>
-			<div css={orgsWrapperStyle} ref={containerRef}>
+		<Scrollbars
+			style={{
+			maxHeight: 250,
+			paddingRight: 80,
+			marginRight: -80
+			}}
+		>
+			<>
 				{organizations.map((org) => {
-					const hasError = !canConnect(org);
-					const orgDivStyles = hasError
-						? [orgDivStyle, orgDivWithErrorStyle]
-						: [orgDivStyle];
-					return (
-						<div key={org.id} css={orgDivStyles}>
-							{canConnect(org) ? (
-								<>
-									<span css={orgNameStyle}>{org.account.login}</span>
-									{loaderForOrgClicked && clickedOrg?.id === org.id ? (
-										<LoadingButton style={{ width: 80 }} isLoading>
-											Loading button
-										</LoadingButton>
-									) : (
-										<Button
-											isDisabled={
-												loaderForOrgClicked && clickedOrg?.id !== org.id
-											}
-											onClick={async () => {
-												setLoaderForOrgClicked(true);
-												setClickedOrg(org);
-												try {
-													// Calling the create connection function that is passed from the parent
-													await connectingOrg(org);
-												} finally {
-													setLoaderForOrgClicked(false);
-												}
-											}}
-										>
-											Connect
-										</Button>
-									)}
-								</>
-							) : (
-								<>
-									<div>
+						const hasError = !canConnect(org);
```

Comment:
Can we add some tests for this component? Don't think we need anything too crazy but we should at least have some for canConnect with different inputs to ensure it returns the correct result based on the organization's properties (requiresSsoLogin, isIPBlocked, isAdmin).

Reply:
Added test cases for the error conditions.